server/src/main/java/org/elasticsearch/rest/ServerlessApiProtections.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.rest;
+
+public class ServerlessApiProtections {
```

Comment:
Nit: Should we just call it `ApiProtections`?

Reply:
I don't mind. I made it explicit because the enum is `@ServerlessScope`, so it's not intended to ever be a generic API protections facility.